src/js/apps/globals/search_app.js

```diff
@@ -6,13 +6,13 @@ import { PatientSearchModal } from 'js/views/globals/search/patient-search_views
 
 export default App.extend({
   onStart({ prefillText }) {
-    // const settings = Radio.request('bootstrap', 'currentOrg:setting', 'patient_search_settings');
     this.showSearch(prefillText);
   },
   showSearch(prefillText) {
     const patientSearchModal = new PatientSearchModal({
       collection: Radio.request('entities', 'searchPatients:collection'),
       prefillText,
+      settings: Radio.request('bootstrap', 'currentOrg:setting', 'patient_search_settings'),
```

Comment:
Can we retrieve this org setting in the `/views/globals/search/patient_search_views.js` view file instead?

Reply:
we'll need the settings in this app to determine what search stuff will need to be available in future work.  However I don't see why the view couldn't also retrieve them directly.

Reply:
Okay, makes sense! Will keep it in the app for future use.